src/core/acl.cc

```diff
@@ -552,6 +573,37 @@ int32_t acl::process_whitelist(bool &ao_match, rqst_ctx &a_ctx)
                 }
         }
         // -------------------------------------------------
+        // subdivision iso
+        // -------------------------------------------------
+        if(m_sd_iso_whitelist.size() && l_buf &&
+           l_buf_len && a_ctx.m_src_sd1_iso.m_data && a_ctx.m_src_sd1_iso.m_len &&
+           a_ctx.m_geo_cn2.m_data &&
```

Comment:
break up the lines with just one logical AND in the line and at the end. Lets try to be consistent with rest of code base. More style info https://google.github.io/styleguide/cppguide.html#Boolean_Expressions